client/src/@types/api/Title/index.d.ts

```diff
@@ -5,8 +5,7 @@ export interface TitleResponseData {
 		name: string,
 		slug: string,
 		entry_count: number,
-		category_id: string,
-		category_ancestors: string[],
+		tags: string[],
```

Comment:
According to the server implementation, we need to use the tag id to get the tag details. We could rename the `tags` to the `tag_ids`.

If we want to eliminate the multiple API calls to get the tags name, we could create a `SimpleTag` to supply the tag id with name, 
```typescript
const type SimpleTag = {
  __id: string,
  name: string,
}
```

And we could use it here:
```suggestion
		tags: SimpleTag[],
```